app/src/pages/loginPage/loginPage.tsx

```diff
@@ -32,7 +32,7 @@ export const LoginPage = (props: ILoginPageProps): React.ReactElement => {
       hasErrors = true;
     }
     if (password && password.length < 8) {
-      setPasswordError('Use 8 or more characters');
+      setPasswordError('Wrong password. Try again or click Forgot password to reset it');
```

Comment:
previous error didn't really make sense. Not sure if this error makes sense here anymore either cos we're not calling the backend lol. Maybe this is what the TODO on line 46 should return?